libs/core/src/lib/select/select.module.ts

```diff
@@ -16,10 +15,9 @@ import { BusyIndicatorModule } from '../busy-indicator/busy-indicator.module';
         BusyIndicatorModule,
         CommonModule,
         PopoverModule,
-        MenuModule,
         ButtonModule,
         IconModule,
-        LoadingSpinnerModule
+        ListModule
```

Comment:
Isn't this too big overhead to import module with all its directives ?

Reply:
Well it is. The only alternatives I see are:
- creating a `List->Select` dedicated module
- implementing list features in Select and abandon `ListModule` import
but both of them seems to introduce additional maintenance. This is a thing to discuss with the team if we want to and how we want to handle it.